consoleme/handlers/v2/notifications.py

```diff
@@ -0,0 +1,159 @@
+from typing import List
+
+import sentry_sdk
+
+from consoleme.config import config
+from consoleme.handlers.base import BaseAPIV2Handler
+from consoleme.lib.generic import is_in_group
+from consoleme.lib.notifications.models import (
+    ConsoleMeNotificationUpdateAction,
+    ConsoleMeNotificationUpdateRequest,
+    ConsoleMeUserNotification,
+    GetNotificationsForUserResponse,
+)
+from consoleme.lib.plugins import get_plugin_by_name
+from consoleme.lib.v2.notifications import (
+    fetch_notification,
+    get_notifications_for_user,
+    write_notification,
+)
+from consoleme.models import Status2, WebResponse
+
+stats = get_plugin_by_name(config.get("plugins.metrics", "default_metrics"))()
+log = config.get_logger()
+
+
+class NotificationsHandler(BaseAPIV2Handler):
+    """
+    A web handler for serving, updating, and (in the future) creating notifications. Current notifications are based
+    around policy generation from CloudTrail errors.
+    """
+
+    async def get(self):
+        try:
+            # Check notifications for user, return notifications for user
+            notification_response: GetNotificationsForUserResponse = (
```

Comment:
Nit: this annotation is redundant due to type inference. Not a problem though, because sometimes it's good to be explicit.

Reply:
removed comment